pkg/debug/scripts/collect-info.sh

```diff
@@ -234,6 +236,10 @@ collect_pillar_backtraces()
 
     kill $pid
 
+    # collect memory backtrace
+    eve exec debug curl "http://127.1:6543/debug/pprof/heap?debug=1" > "$DIR/pillar-memory-backtraces"
```

Comment:
Please something like
```cnt=10; while [ "$cnt" -ge 0 ]; do curl -m 5 http://blah:port && break; cnt=$((cnt - 1)); done;```
Then no need to introduce artificial wait by separating http start and actual curl.

Reply:
Is this way ok?